code/modules/mob/living/carbon/human/human_damage.dm

```diff
@@ -31,17 +26,45 @@
 	tod = 0
 	timeofdeath = 0
 
+/mob/living/carbon/human/adjustBrainLoss(var/amount)

```

Comment:
No - brainloss still handles the mob's brain damage. It's just that brainloss now also damages the brain (if someone has 50 brainloss, their brain also has 50 damage). This means it's possible to fix brain damage by repairing the brain through surgery.

Everything plays the same way - it's just that the brain now takes some damage, too.